src/client/src/rt-components/tear-off/DesktopWindow.ts

```diff
@@ -33,7 +37,7 @@ export const openDesktopWindow = (config: DesktopWindowProps) => {
         resolve(win)
       },
       error => {
-        console.log('Error creating window:', error)
+        errorLogger.log('Error creating window:', error)
```

Comment:
It's not significant to indicate `error` — and in general a logger represents a service before a particular state of that service. So, just `logger` is fine.